client/js/components/wobbly-edge.js

```diff
@@ -11,9 +11,14 @@ const wobblyEdge = (el) => {
     options.numberOfPoints = el.getAttribute('data-number-of-points');
   }
 
+  if (el.getAttribute('data-is-static') === 'true') {
+    options.isStatic = true;
+  }
+
   const defaults = {
     maxIntensity: 50,
-    numberOfPoints: 5
+    numberOfPoints: 5,
+    isStatic: false
   };
```

Comment:
Would that rely on each wobbly-edge component having a `data-is-static` attribute? The `Object.assign({},…)` below to mix the defaults in with the options prevents mutation, no?